geode-junit/src/main/java/org/apache/geode/pdx/TestObjectForJSONFormatter.java

```diff
@@ -992,6 +993,34 @@ boolean compareMaps(Map m1, Map m2) {
     return true;
   }
 
+  @Override
+  public int hashCode() {
+    int result =
+        Objects
+            .hash(p_bool, p_byte, p_short, p_int, p_long, p_float, p_double, w_bool, w_byte,
+                w_short,
+                w_int, w_long, w_bigInt, w_float, w_bigDec, w_double, w_string, c_list, c_set,
+                c_queue, c_deque, m_empByCity, day);
```

Comment:
Could this be formatted slightly neater? The following seems to make spotless happy on my machine:
`int result = Objects.hash(p_bool, p_byte, p_short, p_int, p_long, p_float, p_double, w_bool,
        w_byte, w_short, w_int, w_long, w_bigInt, w_float, w_bigDec, w_double, w_string, c_list,
        c_set, c_queue, c_deque, m_empByCity, day);`